notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.            print(ie, "length doesnt match", len(Jet[ie]), len(df[ie].Jet_pt))
</pre><p>What is the percentage of events for which length, ptrel, etc do not match?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Guess this is answered by @colizz comment already, will resolve this conversation